_posts/2018-12-22-35c3.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Void Linux at 35c3"
+comments: true
+---
+
+![35C3 Void](/assets/img/35c3-void.png)
+
+2018 has been an intense year for Void. There were many ups and downs. To wrap
+up the months behind we're meeting at the [35c3](https://events.ccc.de/category/congress/35c3/).
+
+For meeting other Voiders and chat about related and unrelated topics there's
+the
+[**Void Linux Assembly**](https://signup.c3assemblies.de/assembly/cf15ba5d-5bb1-473d-827c-b83862de75a1)
+
+Furthermore, on day 1 (2018-12-27) 19:00, [@Gottox](https://twitter.com/Gottox) is doing his
```

Comment:
Furthermore would usually be a continuation of a singular idea, maybe use 'Also' instead.